conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+        edDraftURI: "https://w3c.github.io/PWETF/",
+        specStatus: "ED",
+        shortName: "pwe-crp",
+        editors: [{
+            name: "Tzviya Siegman",
+            company: "Wiley",
+            companyURL: "https://www.wiley.com",
+            w3cid: 65542
+        }, {
+            name: "Wendy Reid",
+            company: "Rakuten Kobo",
+            companyURL: "https://rakuten.com",
+            w3cid: 102009
+        }],
+        github: {
+            repoURL: "https://github.com/w3c/PWETF/",
+            branch: "main"
+        },
+        };
+        </script>
+    </head>
+    <body>
+        <section id="abstract">
+            <p>This document is a companion to the W3C Code of Conduct which clarifies the procedures for handling incidents relating to Code of Conduct violations or complaints.</p>
+        </section>
+
+        <section id="sotd"></section>
+
+        <section id="introduction">
+            <h2>Introduction</h2>
+
+            <p>As we engage with one another, incidents can happen. Incidents can take many forms, but participants have the right to a safe and positive work environment. The goal of conflict resolution is that participants in W3C can:</p>
+
+            <ul>
+                <li>Get help and support to safely resolve issues that negatively affect them</li>
+                <li>Correct their behaviours that negatively affect other participants in W3C</li>
+                <li>Continue to safely work and contribute to the W3C after an issue has been raised</li>
+            </ul>
+                
+            <p>The Code of Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+                
+            <p>All situations are different and W3C participants are encouraged to use whatever method of conflict resolution they feel comfortable with.</p>
+                
+            <p>Participants are encouraged to try and resolve issues themselves in the first instance. However, if participants do not feel comfortable or safe doing so, they can get help through an ombudsperson, facilitators and/or mediation.</p>
+                
+            <p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally or in the context of the group. Participants should immediately report an incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.</p>
+                
+            <p>Everyone involved in the conflict resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>
```

Comment:
I think we need to be more specific here.  I don't think we want to require everyone to maintain confidentiality all the time.  I think the confidentiality requirement mostly applies to the ombuds and others who are in a position of W3C responsibility for responding to complaints of CoC violations.   Presumably a person who experienced a CoC violation should have the option of reporting it publicly, and anyone who is accused of a CoC violation should have the option of reporting that accusation publicly.

Reply:
Created issue #359 to cover this question.